.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
+charset = utf-8
```

Comment:
Just curious, why limit this default charset to JS/Py?

Reply:
Just stole that from the editorconfig example on their homepage, so had assumed they did it for a reason.  Shall we set it for all files?